frontend/device/sdl/event_map_sdl.lua

```diff
@@ -4,8 +4,9 @@ return {
     [38] = "A", [39] = "S", [40] = "D", [41] = "F", [42] = "G", [43] = "H", [44] = "J", [45] = "K", [46] = "L",
     [52] = "Z", [53] = "X", [54] = "C", [55] = "V", [56] = "B", [57] = "N", [58] = "M",
 
-    [22] = "Back", -- Backspace
-    [36] = "Enter", -- Enter
+    [22] = "Backspace", -- Backspace
+    [36] = "Press", -- Enter
+    [41] = "Back", -- Escape
```

Comment:
Sorry, that's my bad. I intended the 41 only for SDL2.

Reply:
Yes, I see, so how to find the keycodes ?

Reply:
The program just spits them out in the console, but the SDL also has plenty of info like https://wiki.libsdl.org/SDLScancodeLookup

For SDL1.2 I don't really know. The state of SDL1.2 is "it probably still works but I doubt anyone's even tried it in years."

For SDL2 the relevant headers are in /usr/include/SDL2/SDL_scancode.h. It's probably something similar for 1.2.

Reply:
Looks like this stuff worked completely different in SDL1.2: https://www.libsdl.org/release/SDL-1.2.15/docs/html/guideinputkeyboard.html

So sorry, I have no idea. I'd say just leave it unless it really bothers you. ;-)